mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -56,6 +56,40 @@ func TestAccContainerCluster_basic(t *testing.T) {
 	})
 }
 
+func TestAccContainerCluster_resourceManagerTags(t *testing.T) {
```

Comment:
Can you add an update test? Where we use the functionality for update added here? Specifically we want to test out a couple operations:
1. Adding a tag
2. Removing a tag
3. Removing all tags

Reply:
Hello @slevenick, there is not current in-place support for these operations but for node_pools and node_auto_config

Reply:
Then the field should have ForceNew: true I think?

Reply:
there is a mid function that forces specifically for cluster resource